content/docs/reference/config/project-descriptor.md

```diff
@@ -76,6 +76,18 @@ The schema for the `project descriptor` is:
       - **`uri`** _(string, default=`urn:buildpack:<id>`)_\
         A URL or path to an [archive][supported-archives], a packaged buildpack (saved as a `.cnb` file), or a directory. If path is relative, it must be relative to the `project.toml`.
 
+      - **`script`** _(list, optional)_
+      Defines an inline buildpack.
+
+        - **`api`** _(string, required, current: `0.5`)_\
+          The Buildpack API version the buildpack adheres to. Used to ensure [compatibility][api-compat] against the [lifecycle][lifecycle].
```

Comment:
If I'm not mistaken this link to the `lifecycle` is broken (no ref).

Reply:
Fixed in 341c23c, but I'm not sure why this didn't turn up in `make check-links` (for me locally at least).